app/views/catalog/_show_manuscript.html.erb

```diff
@@ -33,3 +32,8 @@
     <% end -%>
   </dl>
 </div>
+<button type='button' class='btn btn-default btn-details hidden' data-toggle='collapse' data-target='#metadataDetails' aria-expanded='false' aria-controls='metadataDetails'>
+  <span class='btn-text-show'><%= I18n.t('catalog.show_manuscript.show_details') %></span>
+  <span class='btn-text-hide hidden'><%= I18n.t('catalog.show_manuscript.hide_details') %></span>
```

Comment:
Since we're using the partial path as part of the `I18n` key path, we can shorthand these keys to be just `t('.hide_details')`.

This is because if an `i18n` key begins with a `.` `I18n` will construct the full key for us based on the partial path [like above](https://github.com/sul-dlss/vatican_exhibits/pull/168/files#diff-0df38ef53560bd5fdf9b0a14bc490671L3).

Reply:
🤔 I'm also wondering if could take advantage of the button's class to do some of the arrow/text element displaying/hiding.  I believe that the `collapsed` class is added to the button when the element it controls is collapsed.  Maybe we can avoid some of the javascript by `display:none`-ing the `<span>`s when appropriate.

Happy to talk through this if I'm not being clear.

Reply:
Neat! I moved more of the logic over to the SCSS but kept some of the JS fallback stuff. Let me know how that looks. Thanks, @jkeck.